kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -2713,4 +2713,11 @@ object KyuubiConf {
       .version("1.7.0")
       .timeConf
       .createWithDefault(Duration.ofSeconds(60).toMillis)
+
+  val ENGINE_SPARK_LIST_TABLES: ConfigEntry[Boolean] =
+    buildConf("kyuubi.engine.spark.list.tables")
+      .doc("Only query table identifiers when set to true. Work on Spark 3.x only.")
```

Comment:
please update the description to match the configuration name, and we don't need to mention Spark 3.x since Kyuubi only support Spark 3.1 and above